ledger/archival_test.go

```diff
@@ -165,6 +167,12 @@ func TestArchival(t *testing.T) {
 func TestArchivalRestart(t *testing.T) {
 	// Start in archival mode, add 2K blocks, restart, ensure all blocks are there
 
+	// disable deadlock checking code
+	deadlock.Opts.Disable = true
```

Comment:
this might be more proper to save off this disable state to a local variable and restore it at the end (but this is probably fine)

Reply:
good idea. I will do that in a subsequent PR.